Gemfile

```diff
@@ -7,6 +7,7 @@ gem "gds-api-adapters"
 gem "govuk_app_config"
 gem "httparty"
 gem "pg"
+gem "sprockets"
```

Comment:
I think we might be able to drop this commit. Sprockets is a gem that manages the pipeline to build CSS and JS assets. (You'll find those in most of our apps, but this one doesn't have any because it's just an API)